mapswipe_workers/mapswipe_workers/project_types/tile_map_service_grid/project.py

```diff
@@ -58,15 +59,17 @@ def validate_geometries(self):
             raise CustomError(f"Empty file. ")
 
         # check if more than 1 geometry is provided
-        elif layer.GetFeatureCount() > 1:
+        elif layer.GetFeatureCount() > 10:
```

Comment:
should we set this as as definition? max number of features in input geometry

Reply:
@Hagellach37 where can I set this as a definition?

Reply:
you can add this to the `definitions.py` file and then import in this file